src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { Dispatch, SetStateAction } from 'react'
+import { AppRoutes, Route } from '../../views/router/routes'
+import styles from './navbar.module.scss'
+type NavbarProps = {
+	setActiveTab: Dispatch<SetStateAction<Route>>
+	appRoutes: AppRoutes
+}
+
+const Navbar: React.FC<NavbarProps> = ({ setActiveTab, appRoutes }) => {
+	return (
+		<>
+			<nav className={styles['nav-menu']}>
+				{Object.values(appRoutes).map((route, index) => (
+					<button
+						key={index}
+						className={styles['navbar-button']}
+						onClick={() => setActiveTab(route)}
+					>
+						{route.icon}
+					</button>
+				))}
+			</nav>
```

Comment:
With the changes outline in `routes.ts` (`.tsx`), we just need to change how `routes.icon` is called.

I have two suggestions, code style preference is the only difference:

```suggestion
			<nav className={styles['nav-menu']}>
				{Object.values(appRoutes).map((route, index) => (
					<button
						key={index}
						className={styles['navbar-button']}
						onClick={() => setActiveTab(route)}
					>
						<route.icon />
					</button>
				))}
			</nav>
```
 OR, extract the icon into a named variable if you want the format of the component name to match the other component names
```suggestion
			<nav className={styles['nav-menu']}>
				{Object.values(appRoutes).map((route, index) => {
					const Icon = route.icon

					return (
						<button
							key={index}
							className={styles['navbar-button']}
							onClick={() => setActiveTab(route)}
						>
							<Icon />
						</button>
					)
				})}
			</nav>
```

Reply:
Of note (in case this JS behavior isn't familiar)
``` Typescript
() => (
    <div>
        <MyComponent />
    <div />
)
```
is equivalent shorthand (i.e. equivalent) to
``` Typescript
() => {

    return (
        <div>
            <MyComponent />
        <div />
    )
}
```